mitodl/micromasters
# Add support for congratulation letters for non-fa programs

#### What are the relevant tickets?
#4247 

#### What's this PR do?
Add support for congratulation letter for non-fa programs

#### Any background context you want to provide?
Read RFC at #4258 

**How to test:**

- In the CMS admin create `ProgramCertificateSignatories`, `program_letter_text` and `program_letter_logo` for the program.
- Set `program.financial_aid_availability = False`
- Enroll user in program.
- Create a passed course with a grade by running `atler_data` command with `set_past_run_to_passed` for all courses in enrolled program.
- Create course certificate for all courses:
- `MicromastersCourseCertificate.objects.create(user=<user>, course=<course>)`
- Congratulations letter should automatically created after having certificate in all courses.


#### Screenshots (if appropriate)

This is how it will looks like.

**Dashboard:**
![screen shot 2019-03-04 at 5 59 24 pm](https://user-images.githubusercontent.com/4245618/53734970-52230c80-3ea7-11e9-82e1-7fb78f058fce.png)

**Congratulations Letter:**
![screen shot 2019-02-28 at 4 18 08 pm](https://user-images.githubusercontent.com/4245618/53562932-7b286200-3b74-11e9-885b-e8f9229d4936.png)
